internal/k8sutils/pods.go

```diff
@@ -12,3 +18,30 @@ func PodIsReady(pod *corev1.Pod) bool {
 	}
 	return false
 }
+
+// PodHash returns a hash value calculated from Pod spec and
+// a collisionCount to avoid hash collision.
```

Comment:
seems this comment is not accurate since collisionCount code is commented out.

Reply:
We can uncomment the collision code within func

Reply:
I removed collsionCount as parameter since it's not used and not passed anywhere.